packages/datepickers/demo/datepicker.stories.mdx

```diff
@@ -25,6 +25,16 @@ import README from '../README.md';
         control: 'radio',
         options: DATE_STYLE_OPTIONS,
         table: { category: 'Story' }
+      },
+      message: { name: 'children', control: { type: 'text' }, table: { category: 'Message' } },
+      validation: {
+        options: ['success', 'warning', 'error'],
+        control: { type: 'radio' },
+        table: { category: 'Message' }
+      },
```

Comment:
Let's restructure this to behave more like [Input](https://zendeskgarden.github.io/react-components/?path=/story/packages-forms-input--uncontrolled) or [Combobox](https://zendeskgarden.github.io/react-components/?path=/story/packages-dropdowns-combobox--combobox)
- default the message `children` to "Message"
- add a **Story** category section control for toggling the `Message` on
- move `validation` to an **Input** category section and allow it to control validation styling for both the input and the message together (as there is no design recommendation for seeing them happen separately)